### PR TITLE
Updated to align with v3.7 versions

### PR DESCRIPTION
**Pull request by oybed**
#### What does this PR do?
Updating to better handle v3.7 installations and better handle DNS aspects + now using `ansible galaxy` for `openshift-ansible` dependencies

This PR is dependent on the following PRs being merged:
https://github.com/openshift/openshift-ansible-contrib/pull/882
https://github.com/openshift/openshift-ansible-contrib/pull/886
... in addition a new `v3.7.0` release should be cut of the `openshift-ansible-contrib` repo to support the dependency. 

#### How should this be manually tested?
Follow the instructions in the README(s) to install an OpenShift cluster on OpenStack 

#### Is there a relevant Issue open for this?
N/A
#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

**Comment by oybed** [comment]
@etsauer please review - this is now ready for merge. Note that per our v3.7 plan; existing DNS servers are required for this to be successful. Additional documentation should be created for this and submitted.

File: README.md [hunk 1]
```diff
@@ -28,15 +28,3 @@ For some tasks, the CASL repository has several dependencies on external reposit
 
 > **Note:** The dependencies are managed using `ansible-galaxy` and the specific instructions will call this out when there is a need to use galaxy to pull in the correct dependencies.
 
-Below is a list of versions of `openshift-ansible` the CASL team has leveraged and found to be stable/working. Note that other versions may work as well. Feel free to submit PRs for this file with updated versions as they are found to be functional.
```
Comment (etsauer) [inline README.md]: @oybed where are we moving this to? I only see a delete.

Comment (oybed) [reply README.md]: @etsauer this is now being handled with `ansible-galaxy`, so we don't track anymore ... just use the latest `release-3.x` version:
https://github.com/oybed/casl-ansible/blob/v37/casl-requirements.yml#L12-L14

Comment (oybed) [reply README.md]: It also simplifies the documentation as the users don't have to manually set up `openshift-ansible` anymore

**Comment by etsauer** [comment]
ok, them LGTM. Merging

**Pull request merged**